### DISCUSSION TITLE
Multi AI models in monailabel/bundle on 3D Slicer

### DISCUSSION THREAD

**Discussion by junxiant** [Project-MONAI/MONAI#7834] [Q&A] [answered]
Hello there,

Following this tutorial i was able to reproduce the lung nodule detection on 3D slicer. https://github.com/Project-MONAI/tutorials/blob/main/monailabel/monailabel_monaibundle_3dslicer_lung_nodule_detection.ipynb

I saw in the tutorial that there were two datasets, luna (detection) and msd (segmentation). Then I was looking into multi-ai models, meaning first the lung nodule detection model will run, then second it will run the lung nodule segmentation, producing a bounding box prediction and the segmentation label, which can be displayed on the 3D Slicer. Is this even possible? 

I know that multi ai models for segmentation tasks (two segmentation models) would work as shown in this multi-ai deploy app tutorial https://docs.monai.io/projects/monai-deploy-app-sdk/en/stable/notebooks/tutorials/05_multi_model_app.html but i am trying to do [object detection -> segmentation]

link to discussion in monailabel repo https://github.com/Project-MONAI/MONAILabel/discussions/1707#discussioncomment-9792495

**Comment by KumoLiu** [comment, marked as answer]
Get replied in https://github.com/Project-MONAI/MONAILabel/discussions/1707#discussioncomment-9792495.
